tm2/pkg/bft/state/eventstore/types/config.go

```diff
@@ -7,8 +7,8 @@ type EventStoreParams map[string]any
 
 // Config defines the specific event store configuration
 type Config struct {
-	EventStoreType string
-	Params         EventStoreParams
+	EventStoreType string           `toml:"event_store_type" comment:"Type of event store"`
+	Params         EventStoreParams `toml:"event_store_params" comment:"Event store parameters"`
```

Comment:
This is probably fine, but since it's a breaking change (`EventStoreType` -> `event_store_type` in cfg), mark this in the title/description?

Reply:
I've went through and verified this shouldn't cause old configuration files to break:

`EventStore` types and params are now saved in the configuration file (they were previously not present in the template, so unsaved). They were, however, always initialized as part of the default config, just never saved on disk